rn/Teacher/src/modules/pages/edit/PageEdit.js

```diff
@@ -17,7 +17,7 @@
 // @flow
 
 import i18n from 'format-message'
-import * as React from 'react'
+import React, { Component } from 'react'
```

Comment:
Does the `* as React` also bring in types, where the default export doesn't? I can't remember what the difference is.

Reply:
I am not sure if types are brought in with `* as React`. I swapped to `*` since I was using `Fragment` as well. So since I'm not I figured I would revert it back.

Reply:
👍